### DISCUSSION TITLE
Hooks fails when using signals through unkpkg

### DISCUSSION THREAD

**Discussion by marlun** [preactjs/signals#287] [Q&A]
I'm trying to use Preact with htm and signals without any build step and importing it through unpkg:

```javascript
import {
  computed,
  signal,
} from "https://unpkg.com/@preact/signals@latest/dist/signals.module.js?module";
```

But it fails on `https://unpkg.com/preact@10.11.3/hooks?module` giving a 404 error:

<img width="410" alt="image" src="https://user-images.githubusercontent.com/19663/209483095-0da48a0b-0225-4f0c-a077-eaf83056e8fa.png">

**Comment by rschristian** [comment]
I'd probably recommend against using Unpkg these days without good reason. It has a fair number of [longstanding issues](https://github.com/preactjs/preact/issues/2564#issuecomment-636201423) that seem unlikely to ever be resolved.

[Skypack](https://skypack.dev) and [esm.sh](https://esm.sh) are both good alternatives.
